.eslintrc

```diff
@@ -352,6 +352,14 @@
         "wp": "readonly"
       }
     },
+    {
+      "files": ["packages/e2e-test-utils/src/*.js"],
+      "rules": {
+        "eslint-comments/no-unused-disable": "off",
+        "jest/no-hooks": "off",
+        "no-undef": "off"
+      }
+    },
```

Comment:
Why was this added?

Reply:
This linter didn't like page / expected.

Reply:
The correct way of fixing that would be to extend the `files` array here to include `packages/e2e-test-utils/src/**/*.js`:

https://github.com/google/web-stories-wp/blob/446bbd5f30075f8f74884a1f77894749a49b746c/.eslintrc#L333-L354